From 82e49f7c3a5aecf1ffd31c8db1e94c66033a0b03 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 12 Nov 2020 23:47:21 +0200
Subject: [PATCH] [SW-12757] gaudi: security: reuse specs defines

Reuse the HBW SRAM RR defines from specs.
The values have moved to specs because they will be used by other
projects as well.

Change-Id: If0572c9eba97d1ad4e5508bd0e84364e60d7e0af
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index a752def..04d0b0e 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1253,10 +1253,12 @@
 	 * TODO: The below (both SRAM parts) are commented out temporarily
 	 * until the dynamic SRAM RR protection is in place.
 	 */
-/*	preboot_secure_hbw_rr(HBW_RR_SRAM_BGN, 0x7FF0000000, 0x3FFFFF000000);
+/*	preboot_secure_hbw_rr(HBW_RR_SRAM_BGN,
+			      HBW_RR_SRAM_BGN_BASE, HBW_RR_SRAM_BGN_MASK);
 */
 	/* SRAM RR 0x7F_F100_0000 - 0x7F_F140_0000 (4MB) */
-/*	preboot_secure_hbw_rr(HBW_RR_SRAM_END, 0x7FF1000000, 0x3FFFFFC00000);
+/*	preboot_secure_hbw_rr(HBW_RR_SRAM_END,
+			      HBW_RR_SRAM_END_BASE, HBW_RR_SRAM_END_MASK);
 */
 	/* SRAM Reduct. RR 0x7F_F140_0000 - 0x7F_F180_0000 (4MB) */
 	preboot_secure_hbw_rr(HBW_RR_SRAM_R_BGN, 0x7FF1400000, 0x3FFFFFC00000);
